new_guidance_commands/CMakeLists.txt

```diff
@@ -0,0 +1,201 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(new_guidance_commands)
```

Comment:
Minor pedantry, but I'd rename this to Command Repeater or something along those lines. "New Guidance Commands" requires a bit more context to understand, why are we using both GuidanceCommands and New Guidance Commands, what's different about the new one, etc.